doc/src/manual/noteworthy-differences.md

```diff
@@ -167,13 +169,14 @@ For users coming to Julia from R, these are some noteworthy differences:
   * Julia's [`sum`](@ref), [`prod`](@ref), [`maximum`](@ref), and [`minimum`](@ref) are different
     from their counterparts in R. They all accept one or two arguments. The first argument is an iterable
     collection such as an array.  If there is a second argument, then this argument indicates the
-    dimensions, over which the operation is carried out.  For instance, let `A=[[1 2],[3 4]]` in Julia
-    and `B=rbind(c(1,2),c(3,4))` be the same matrix in R.  Then `sum(A)` gives the same result as
-    `sum(B)`, but `sum(A, 1)` is a row vector containing the sum over each column and `sum(A, 2)`
+    dimensions, over which the operation is carried out.  For instance, let `A=[1 2; 3 4]` in Julia
```

Comment:
Could add a space around the Julia `=` as you've done with the R `<-` assignment.

Reply:
fixed